libnd4j/include/ops/declarable/impl/PlatformHelper.cpp

```diff
@@ -76,8 +76,8 @@ sd::NDArray* PlatformHelper::getZ(graph::Context& ctx, int inputId) {
         sd_printf("Can't get Z variable for node_%i!\n", ctx.nodeId());
       }
     } else {
-      sd_printf("BOOM!\n", "");
-      throw std::runtime_error("Boom!");
+      sd_printf("Failed execution!\n", "");
+      throw std::runtime_error("Failed execution");
```

Comment:
While "Failed execution" is a better message than "Boom", I wonder if we can't have something more descriptive here?

Also, Do we need to both print and throw an exception?